sensor_msgs/Image ima
int64 a
---
float32[] desc
string model_type
